Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -376,6 +369,54 @@ CGImageRef CGImageCreateWithMaskingColors(CGImageRef image, const CGFloat* compo
 
 #pragma region WIC_HELPERS
 
+DisplayTexture* _CGImageGetDisplayTexture(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    CGIWICBitmap* iwicImg = dynamic_cast<CGIWICBitmap*>(image->ImageSource().Get());
```

Comment:
Yes, but given it's a COM object. I've updated how this gets resolved.

---
In reply to: [86919393](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r86919393) [](ancestors = 86919393)